src/physics/circular_filament.rs

```diff
@@ -1,19 +1,88 @@
+/// * `zprime`:  (m) z-coord of each observation point, length `n`
+/// * `out_psi`: (Wb) or (H) or (T-m^2) or (V-s), poloidal flux at observation locations, length `n`
+///
+/// # Commentary
+///
+/// Represents contribution from a current at (R, Z) to an observation point at (Rprime, Zprime)
+///
+/// Note Jardin's 4.61-4.66 presents it with a different definition of
+/// the elliptic integrals from what is used here and in scipy.
+///
+/// # References
+///
+///   \[1\] D. Kaltsas, A. Kuiroukidis, and G. Throumoulopoulos, “A tokamak pertinent analytic equilibrium with plasma flow of arbitrary direction,”
+///         Physics of Plasmas, vol. 26, p. 124501, Dec. 2019,
+///         doi: [10.1063/1.5120341](https://doi.org/10.1063/1.5120341).
+///
+///   \[2\] S. Jardin, *Computational Methods in Plasma Physics*, 1st ed. USA: CRC Press, Inc., 2010.
+///
+///   \[3\] J. Huang and J. Menard, “Development of an Auto-Convergent Free-Boundary Axisymmetric Equilibrium Solver,”
+///         Journal of Undergraduate Research, vol. 6, Jan. 2006, Accessed: May 05, 2021. \[Online\].
+///         Available: <https://www.osti.gov/biblio/1051805-development-auto-convergent-free-boundary-axisymmetric-equilibrium-solver>
+///
+///   \[4\] J. C. Simpson, J. E. Lane, C. D. Immer, R. C. Youngquist, and T. Steinrock,
+///         “Simple Analytic Expressions for the Magnetic Field of a Circular Current Loop,”
+///         Jan. 01, 2001. Accessed: Sep. 06, 2022. [Online]. Available: <https://ntrs.nasa.gov/citations/20010038494>
+pub fn flux_circular_filament_par(
+    ifil: &[f64],
+    rfil: &[f64],
+    zfil: &[f64],
+    rprime: &[f64],
+    zprime: &[f64],
+    out_psi: &mut [f64],
+) -> Result<(), &'static str> {
+    // Chunk inputs
+    let ncores = std::thread::available_parallelism()
+        .unwrap_or(NonZeroUsize::MIN)
+        .get();
+
+    let n = (rprime.len() / ncores).max(1);
+
+    let rprimec = rprime.par_chunks(n);
+    let zprimec = zprime.par_chunks(n);
+
+    let outc = out_psi.par_chunks_mut(n);
+
+    // Run calcs
+    outc.zip(rprimec.zip(zprimec))
+        .try_for_each(|(outc, (rc, zc))| flux_circular_filament(ifil, rfil, zfil, rc, zc, outc))?;
+
+    Ok(())
+}
+
 /// Flux contributions from some circular filaments to some observation points, which happens to be
 /// the Green's function for the Grad-Shafranov elliptic operator, $\Delta^{\*}$.
 ///
 /// # Arguments
 ///
-/// * `current`: (A) current in each filament, length `m`
+/// * `ifil`: (A) current in each filament, length `m`
 /// * `rfil`:    (m) r-coord of each filament, length `m`
 /// * `zfil`:    (m) z-coord of each filament, length `m`
 /// * `rprime`:  (m) r-coord of each observation point, length `n`
 /// * `zprime`:  (m) z-coord of each observation point, length `n`
-/// * `out_psi`:     (Wb) or (T-m^2) or (V-s), poloidal flux at observation locations, length `n`
+/// * `out_psi`: (Wb) or (H) or (T-m^2) or (V-s), poloidal flux at observation locations, length `n`
```

Comment:
A weber is a henry ampere, not a henry.
```suggestion
/// * `out_psi`: (Wb) or (H-A) or (T-m^2) or (V-s), poloidal flux at observation locations, length `n`
```